tools-local/unix/probe-tool.sh

```diff
@@ -2,13 +2,17 @@
 
 usage()
 {
+    echo ""
```

Comment:
Why echo an empty line at the beginning?  Seems inconsistent with other usage messages.